src/Test/Perf/infra/install_vsixes.csx

```diff
@@ -25,6 +25,10 @@ var vsixes = new[]
 
 var installer = Path.Combine(binariesDirectory, "VSIXExpInstaller.exe");
 
+// Uninstall all previously installed VSIXes
+TestUtilities.ShellOutVital(installer, $"/rootSuffix:RoslynPerf /uninstallAll", IsVerbose(), logger, binariesDirectory);
```

Comment:
I was thinking about it initially but since this is part of the automation chain and it will be RoslynPerf only for now, I let it be. If we come across a situation to change it, we shall extract it then.